tests/CarbonInterval/ForHumansTest.php

```diff
@@ -245,4 +245,74 @@ public function testOptionsAsArray()
             'options' => CarbonInterface::ONE_DAY_WORDS,
         ]));
     }
+
+    public function testRoundYears()
+    {
+        CarbonInterval::setLocale('en');
+        $interval = CarbonInterval::years(2)->months(11);
+        $this->assertEquals('3 years', $interval->forHumans(['parts' => 1, 'options' => CarbonInterface::ROUND]));
+    }
+
+    public function testRoundUp()
+    {
+        CarbonInterval::setLocale('en');
+        $interval = CarbonInterval::days(2)->hours(23);
+        $this->assertEquals('3 days', $interval->forHumans(['parts' => 1, 'options' => CarbonInterface::ROUND]));
+    }
+
+    public function testRoundDown()
+    {
+        CarbonInterval::setLocale('en');
+        $interval = CarbonInterval::days(2)->hours(11);
+        $this->assertEquals('2 days', $interval->forHumans(['parts' => 1, 'options' => CarbonInterface::ROUND]));
+    }
```

Comment:
Can we have a round down and up for each unit, so we check the dividers are correct and don't change later by accident.

Plus, it will fulfill the coverage: https://codecov.io/gh/briannesbitt/Carbon/compare/e01ecc0b71168febb52ae1fdc1cfcc95428e604e...898f3582ab324874f48bbf5a8f60a97e4e2c755b/diff

Mostly I'm interesting in seeing what rounding happens when I have a month+week interval or a year+week interval.